### PR TITLE
feat: Add BI webview and secret grant type

### PR DESCRIPTION
**Pull request by doubleface**
This is needed to handle bi connection creation via [BI webviews](https://docs.budget-insight.com/reference/webview)

### CODE REVIEW

**Comment by nono** [comment]
Don't forget the documentation

File: docs/konnectors-workflow.md [hunk 1]
```diff
@@ -690,6 +704,23 @@ Location: https://domain.biapi.pro/2.0/auth/webview/reconnect
...
+It is possible to use this workflow for konnectors by registering an account
+type with the following parameter:
+
+- `grant_mode`, with `bi_webview+secret` as the value
+- `redirect_uri`, with an URL like `https://oauthcallback.mycozy.cloud/accounts/paypal/redirect`
+- `client_id`, with the client ID given by Budget Insight
+- `auth_endpoint`, with `https://{domain}.biapi.pro/2.0/auth/webview/fr/connect` (with the correct `domain`)
+- `reconnect_endpoint`, with `https://{domain}.biapi.pro/2.0/auth/webview/reconnect` (idem).
+
+
```
Comment (nono) [inline docs/konnectors-workflow.md:723]: It looks like a copy/paste error: the same block has been added twice. Can you fix it?

File: docs/konnectors-workflow.md [hunk 2]
```diff
@@ -665,6 +665,20 @@ type with the following parameter:
...
+This is a workflow similar to OAuth called
+[webview](https://docs.budget-insight.com/reference/webview)
+
+It is possible to use this workflow for konnectors by registering an account
+type with the following parameter:
+
+- `grant_mode`, with `bi_webview+secret` as the value
+- `redirect_uri`, with an URL like `https://oauthcallback.mycozy.cloud/accounts/paypal/redirect`
+- `client_id`, with the client ID given by Budget Insight
+- `auth_endpoint`, with `https://{domain}.biapi.pro/2.0/auth/webview/fr/connect` (with the correct `domain`)
```
Comment (nono) [inline docs/konnectors-workflow.md:681]: :point_down: Is the `reconnect_endpoint` for `bi_webauth`, `bi_webview`, or both?

Comment (doubleface) [reply docs/konnectors-workflow.md:681]: Both

Comment (nono) [reply docs/konnectors-workflow.md:681]: So, we should keep this line in `Webauth` section, right?

Comment (doubleface) [reply docs/konnectors-workflow.md:682]: Désolé je n'avais pas compris tout de suite. La ligne reconnect_endpoint est de nouveau dans la section webauth.

**Pull request merged**